### DISCUSSION TITLE
Access the mapping key from downstream op when using dynamic mapping

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#8420] [Q&A] [answered]
When using dynamic mapping, is it possible to access the mapping key from the downstream op or is that internal to Dagster?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1639078149047300?thread_ts=1639078149.047300&cid=C01U954MEER).*

**Comment by yuhan** [comment, marked as answer]
You can get the mapping key from the op context, for example, `context.get_mapping_key()`. 
More details in the apidocs: https://docs.dagster.io/_apidocs/execution#contexts
